source/common/http/filter/gzip_filter.cc

```diff
@@ -0,0 +1,274 @@
+#include "common/http/filter/gzip_filter.h"
+
+#include "common/common/macros.h"
+
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_split.h"
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Http {
+
+namespace {
+// Default zlib memory level.
+const uint64_t DefaultMemoryLevel{5};
```

Comment:
did you indicate earlier you would change these to  = 5 etc?

Reply:
Do you mean in the docs? Yes.

https://github.com/envoyproxy/data-plane-api/blob/e6aa0cfb337fad4628a37d3f84c5540ce0d6fac0/envoy/api/v2/filter/http/gzip.proto#L14

Reply:
I meant in the code.